Documentation/cluster-crd.md

```diff
@@ -177,6 +178,45 @@ spec:
       deviceFilter: "^sd."
 ```
 
+### Storage Configuration: Cluster wide Directories
+
+This example is based up on the [Storage Configuration: Specific devices](#storage-configuration-specific-devices).
+Individual nodes can override the cluster wiede specified directories list.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook
+---
+apiVersion: rook.io/v1alpha1
+kind: Cluster
+metadata:
+  name: rook
+  namespace: rook
+spec:
+  versionTag: master
+  dataDirHostPath: /var/lib/rook
+  # cluster level storage configuration and selection
+  storage:
+    useAllNodes: false
+    useAllDevices: false
+    deviceFilter:
```

Comment:
to simplify the sample, you might remove the empty properties that aren't used in this sample